### DISCUSSION TITLE
How to select co2 Scenarios in settings file?

### DISCUSSION THREAD

**Discussion by whq552800** [iiasa/CWatM#152] [Q&A]
I'm using CWatM to calculate future crop yield,and when i check the code about ET, I found the co2 scenarios could be selected ,but i can't find where to set in settings file.
![image](https://github.com/user-attachments/assets/b6b43c18-819c-4c05-854e-0986f71a3131)

**Comment by dof1985** [comment]
Hi @whq552800 

The CO2 scenarios is set in the setting files by the setting: 'co2conc', e.g., co2conc = 'path_to_co2_scenarion_file'

You can get files for the RCP 2.6, 4.5, 6.0, and 8.5, here: https://github.com/iiasa/CWatM-Earth-30min/tree/main/RCP_CO2
